dts/arm64/intel/intel_socfpga_agilex.dtsi

```diff
@@ -19,6 +19,24 @@
 			compatible = "arm,cortex-a53";
 			reg = <0>;
 		};
+
+		cpu1: cpu@1 {
```

Comment:
can you add the `cpu0` label as well while you are at this?

Reply:
will do!